src/txreconciliation.cpp

```diff
@@ -27,6 +34,14 @@ constexpr uint32_t MAX_OUTBOUND_FLOOD_TO = 8;
  */
 constexpr std::chrono::microseconds RECON_REQUEST_INTERVAL{16s};
 
+/**
+ * Represents phase of the current reconciliation round with a peer.
+ */
+enum ReconciliationPhase {
```

Comment:
476ced6df27c13fd8d1dae46d09b91549977cac1

nit, enum class and use as `ReconciliationPhase::NONE`?